test/tasks/test_gapic.py

```diff
@@ -23,9 +24,19 @@
 from artman.tasks import gapic_tasks
 
 
+def assert_calls_equal(actual_calls, expected_calls):
+    actual_call_strs = []
+    for call in actual_calls:
+        _, args, _ = call
+        cmd = ' '.join(args[0])
+        actual_call_strs.append(cmd)
+    assert expected_calls == actual_call_strs
+
+
 class GapicConfigGenTaskTests(unittest.TestCase):
     @mock.patch.object(gapic_tasks.GapicConfigGenTask, 'exec_command')
-    def test_execute(self, exec_command):
+    @mock.patch.object(subprocess, 'check_output')
+    def test_execute(self, exec_command, check_output):
```

Comment:
Are you sure these are in the right order? (Python applies decorators bottom-to-top.)

Reply:
No I'm not sure... in fact I was really confused why the values seemed to be swapped. This makes a lot more sense! (Well, the bottom-to-top behavior doesn't make sense to me, that is very confusing.)

Reply:
(This question applies throughout.)